### PR TITLE
Bug 1886572: [on-prem] set keepalived ingress priority to variable

### PR DESCRIPTION
**Pull request by yboaron**
Ingress VIP should be set only on a node that runs an instance of
the default ingress controller pod.
In current code, in case extra ingress-controllers are created the
ingress VIP might be wrongly set on a node that doesn't run an instance of
the default ingress controller.

This PR sets keepalived ingress priority to a variable, the variable should be calculated and set by 
baremetal-runtimecfg based on the presence of the router pod in the node.

This PR should be merged after  - https://github.com/openshift/baremetal-runtimecfg/pull/141

### CODE REVIEW

**Comment by yboaron** [comment]
/hold 
until https://github.com/openshift/baremetal-runtimecfg/pull/141 is merged

**Comment by cybertron** [comment]
/cc @mandre @Gal-Zaidman @jcpowermac @patrickdillon
/lgtm

Not running the other platform tests yet because this won't pass until https://github.com/openshift/baremetal-runtimecfg/pull/141 merges.

**Comment by kikisdeliveryservice** [comment]
@cybertron openshift/baremetal-runtimecfg#141 merged, feel free to run other tests :)

**Review by mandre — COMMENTED** [review]
/lgtm

**Comment by yboaron** [comment]
/hold cancel

 https://github.com/openshift/baremetal-runtimecfg/pull/141  merged

**Comment by cybertron** [comment]
@Gal-Zaidman @jcpowermac Can you take a quick look? It would be nice to get this in for 4.8 since we already merged the first half of the fix.

The two remaining ci failures appear to be failing consistently so I'm not going to try to get a pass from them.

**Review by kikisdeliveryservice — APPROVED** [review]
Adding an approval and a hold - @cybertron @yboaron  remove the hold when you are ready to merge. :)

/hold

**Comment by cybertron** [comment]
/hold cancel

We have lgtm from 3/4 affected platforms and ci is passing everywhere. We should be good to merge this.

**Pull request merged**
